emails/views.py

```diff
@@ -281,21 +287,27 @@ def _sns_message(message_json):
             status=400
         )
 
+    from_address = parseaddr(mail['commonHeaders']['from'][0])[1]
     to_address = parseaddr(mail['commonHeaders']['to'][0])[1]
-    local_portion = to_address.split('@')[0]
-
-    if local_portion == 'noreply':
```

Comment:
We no longer send from `noreply@` as the `Reply-To` value ... we send with `replies@`.

Reply:
We may still want this for users who are trying to respond to the old relayed emails that was sent with `noreply` cal portion.